README.md

```diff
@@ -1,6 +1,33 @@
+# Azure Storage Blob SDK for Go
+The Microsoft Azure Storage SDK for Go allows you to build applications that takes advantage of Azure's scalable cloud storage. 
 
-# Contributing
+This repository contains the open source Blob SDK for Go. Documentation and examples can be found [here](https://godoc.org/github.com/Azure/azure-storage-blob-go/2016-05-31/azblob).
 
+## Features
+* Blob Storage
+	* Create/Read/Update/Delete Block Blobs
```

Comment:
Do you want to add that you can Create/Delete containers too?

Reply:
Done. Included listing operation as well